### PR TITLE
Feature/estimate fix

### PR DESCRIPTION
**Pull request by GEPatSeeker**
I noticed the enum used for estimate was not working so I've amended it to use a struct instead.
Tests have been added and checked.

### CODE REVIEW

**Comment by Morasiu** [comment]
Thanks for PR. I really appreciate all contributions. 

That's looking strange. Enum should be converted from string to enum. I'll look into.

**Comment by Morasiu** [comment]
Hey... I've managed to fix that locally.

image

image

I've used some of your changes:
* Changed property name from `EstimateDto` to `Estimate`
* Changed `EstimateDto.Estimate` type to `string`

Thanks for bringing it to my attention. 

I would be happy to merge your changes if you just go back to using `enum` instead of `struct`. Rest of your changes are very helpful.

**Comment by GEPatSeeker** [comment]
No problem, I've reverted back to 'enum' for you.

File: Clockify.Net/Models/Estimates/EstimateRequest.cs [hunk 1]
```diff
@@ -3,6 +3,6 @@
     public class EstimateRequest
     {
         public int? Estimate { get; set; }
-        public EstimateType Type { get; set; }
+        public string Type { get; set; }
```
Comment (Morasiu) [inline Clockify.Net/Models/Estimates/EstimateRequest.cs:6]:
```suggestion
        public EstimateType Type { get; set; }
```

**Comment by GEPatSeeker** [comment]
Latest changes have broken the tests. I used `enum` for `EstimateDto` but `string` for `Estimate` as otherwise I get `BadRequest` `208`.

**Comment by Morasiu** [comment]
Damn... I'll check that

**Comment by Morasiu** [comment]
I've changed JSON serializer. Everything "should" work fine now.  Only **should** because Clockify removed deleting workspaces from their API and I cannot run all tests.

Anyway. EstimateRequest works now.

@GEPatSeeker  Can you confirm so can I deploy the new version?

**Comment by GEPatSeeker** [comment]
Confirmed, appears resolved, estimate tests are now working.

**Pull request merged**